queries/highlights.scm

```diff
@@ -40,19 +48,19 @@
 ; before ":" or after "."
 ; e.g. record field names, tuple indices, names for named arguments, etc
 (field_access
-  field: (identifier) @property)
+  field: (label) @property)
```

Comment:
Corresponding to the above description, shouldn't we be able to replace all of the special-case "`(label)` inside `(thing)`" with just one `(label) @property`?